product_docs/docs/biganimal/release/using_cluster/05_monitoring_and_logging/third_party_integrations/datadog.mdx

```diff
@@ -0,0 +1,60 @@
+
+## Cost
+
+After enabling the BigAnimal telemetry integration, check your billable Datadog usage and continue to monitor it over time.
+
+Be aware of the following cost considerations:
+
+* *You are responsible for all costs* charged to your Datadog account by telemetry sent by the BigAnimal Datadog integration. Charges are based on usage but they could even result from BigAnimal errors or oversights. If you do not accept this responsibility, do not enable the Datadog integration.
+* Datadog bills for each monitored Postgres node (including any replicas) as a "monitored host" in the your Datadog plan. See the [Datadog price list](https://www.datadoghq.com/pricing/list/) for current pricing per monitored host.
+* Datadog *may* bill for each Postgres container and monitoring agent container running on each monitored Postgres node. See the [Datadog price list](https://www.datadoghq.com/pricing/list/) for current pricing per monitored container.
+* Datadog counts some of the metrics sent by BigAnimal as custom metrics. Datadog plans include a limited number of free unique custom metrics dimensions per host. Additional metrics dimensions above the free limit are billable at a rate set in the Datadog price list. See your [Datadog plan price list](https://www.datadoghq.com/pricing/list/) for details. Although BigAnimal's telemetry integration for Datadog tries to limit custom metrics delivery to minimize additional billable Datadog usage, you should still expect some billable custom metrics usage.
+
+To disable the Datadog integration for BigAnimal and ensure no further costs are incurred on your Datadog account, you must revoke the API key provided to BigAnimal. Disabling the integration on the BigAnimal Portal is not sufficient.
+
+Currently, Datadog does not offer any mechanism for configuring a usage limit or billing cap for monitored hosts, monitored containers, or custom metrics. The Datadog [metrics without limits feature](https://docs.datadoghq.com/metrics/metrics-without-limits/) can limit cardinality-based billing for custom metrics, but it enables ingestion-based billing instead, so the overall price may actually be greater. You must actively monitor and be alert to your Datadog usage-based billing using the Datadog-provided [usage metrics](https://docs.datadoghq.com/account_management/billing/usage_metrics/), such as `datadog.estimated_usage.hosts`, `datadog.estimated_usage.containers`, and `datadog.estimated_usage.metrics`.custom.
+
+
+- [Datadog pricing list](https://www.datadoghq.com/pricing/list/)
+- [Custom metrics billing](https://docs.datadoghq.com/account_management/billing/custom_metrics/?tab=countrate#counting-custom-metrics)
+- [Usage metrics billing](https://docs.datadoghq.com/account_management/billing/usage_metrics/)
```

Comment:
I suggest a "see also:" or "See these Datadog pages for more details:"  here but if it's not our style that's fine too